CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Changes to `dash-player` were sponsored by Volkswagen's Center of Excellence for
 
 ---
 
+## Unreleased
+
+## Added
+- [#73](https://github.com/plotly/dash-player/pull/73) Added async Dash Player
```

Comment:
```suggestion
## Changed
- [#73](https://github.com/plotly/dash-player/pull/73) Converted the JS bundle to async loading
```

I'd describe it as a change rather than an addition